hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/ingest/IngestChecker.java

```diff
@@ -193,10 +198,9 @@ public TransactionInfo runAllChecks(
         // 4. Check throttles
         assertThrottlingPreconditions(txInfo, configuration);
         final var hederaConfig = configuration.getConfigData(HederaConfig.class);
-        if (hederaConfig.ingestThrottleEnabled()) {
-            if (synchronizedThrottleAccumulator.shouldThrottle(txInfo, state)) {
-                throw new PreCheckException(BUSY);
-            }
+        if (hederaConfig.ingestThrottleEnabled() && synchronizedThrottleAccumulator.shouldThrottle(txInfo, state)) {
+            workflowMetrics.incrementThrottled(functionality);
+            throw new PreCheckException(BUSY);
```

Comment:
Is it important to differentiate between 'frontend' and 'backend' throttles, or is it sufficient for the requirements to use the same metric for both?

Reply:
Requirements do not include differentiating between frontend and backend throttles